### PR TITLE
RN 36317

### PR DESCRIPTION
**Pull request by MariekeGO**

### CODE REVIEW

File: release-notes/EPM/I-DOCSIS/EPM_6.1.7_I-DOCSIS.md [hunk 1]
```diff
@@ -107,3 +107,15 @@ In addition, the passive tables are now only filled in when the CCAP/collector p
...
+
+A problem with the aggregating actions used to count the number of CMS with other DOCSIS versions could cause some CCAPs to incorrectly report a high number of cable modems with other DOCSIS versions.
+
+To resolve this, a number of changes were implemented to the Generic CM Collector connector:
+
+- CMs that have DOCSIS version 1.0, 1.1, or "Not initialized" will now be considered to have an unknown DOCSIS version.
+- Exception values have been added to several columns in the cable modem table.
+- Instead of the previous table cleanup method, the cable modem table is now cleaned up when the element starts up.
+
+In addition, for the Skyline EPM Platform connector, the *SetEmptyColumns* method will now set exception values for the latitude and longitude parameters.
```
Comment (MarcosMMA) [inline release-notes/EPM/I-DOCSIS/EPM_6.1.7_I-DOCSIS.md:121]: *for the Skyline EPM Platform DOCSIS connector.

File: release-notes/EPM/I-DOCSIS/EPM_6.1.7_I-DOCSIS.md [hunk 2]
```diff
@@ -107,3 +107,15 @@ In addition, the passive tables are now only filled in when the CCAP/collector p
...
+#### Too many cable modems report to have other DOCSIS version [ID_36317]
+
+A problem with the aggregating actions used to count the number of CMS with other DOCSIS versions could cause some CCAPs to incorrectly report a high number of cable modems with other DOCSIS versions.
+
+To resolve this, a number of changes were implemented to the Generic CM Collector connector:
+
+- CMs that have DOCSIS version 1.0, 1.1, or "Not initialized" will now be considered to have an unknown DOCSIS version.
+- Exception values have been added to several columns in the cable modem table.
+- Instead of the previous table cleanup method, the cable modem table is now cleaned up when the element starts up.
+
```
Comment (MarcosMMA) [inline release-notes/EPM/I-DOCSIS/EPM_6.1.7_I-DOCSIS.md:120]: There is also a change done on the Skyline EPM Platform and Skyline EPM Platform DOCSIS connectors :
- Added exception values to several columns of the cable modem overview table.

File: release-notes/EPM/I-DOCSIS/EPM_6.1.7_I-DOCSIS.md [hunk 3]
```diff
@@ -118,4 +118,4 @@ To resolve this, a number of changes were implemented to the Generic CM Collecto
 - Exception values have been added to several columns in the cable modem table.
 - Instead of the previous table cleanup method, the cable modem table is now cleaned up when the element starts up.
```
Comment (MarcosMMA) [inline release-notes/EPM/I-DOCSIS/EPM_6.1.7_I-DOCSIS.md:120]: There is also a change done on the Skyline EPM Platform and Skyline EPM Platform DOCSIS connectors :

Added exception values to several columns of the cable modem overview table.

**Review by MarcosMMA — CHANGES_REQUESTED** [review]
Remark

File: release-notes/EPM/I-DOCSIS/EPM_6.1.7_I-DOCSIS.md (hunk 3 above)
Comment (MariekeGO) [reply release-notes/EPM/I-DOCSIS/EPM_6.1.7_I-DOCSIS.md:120]: updated!

**Pull request merged**